spec/support/form_helpers.rb

```diff
@@ -138,8 +138,8 @@ def fill_in_transaction_form(expectations: true,
     value: "1000.01",
     disbursement_channel: "Money is disbursed through central Ministry of Finance or Treasury",
     currency: "Pound Sterling",
-    provider_organisation: Organisation.first,
-    receiver_organisation: Organisation.first)
+    providing_organisation: OpenStruct.new(name: "Example provider", type: "Government"),
+    receiving_organisation: OpenStruct.new(name: "Example receiver", type: "Private Sector"))
```

Comment:
This did throw me for a second! When you use it later it looks just like the work you are refactoring! :)

Reply:
We could split the object down into 3 fields for each provider and receiver. I felt like there are many parameters allowed and grouping related fields into objects wouldn't be the worst idea, which in turn made the rest of the test require minimal changes.